cmake/Modules/FindMKLDNN.cmake

```diff
@@ -89,7 +89,7 @@ ENDIF(MKL_FOUND)
 
 IF(MKL_FOUND)
   SET(MKL_cmake_included TRUE)
-  SET(MKLDNN_THREADING "OMP:COMP" CACHE STRING "" FORCE)
+  SET(MKLDNN_THREADING "OMP:COMP" CACHE STRING "")
```

Comment:
what is this change? (asking as cmake noob)

Reply:
without FORCE it prefers an already set value of MKLDNN_THREADING, with FORCE it overwrites it. I'm a cmake noob myself, so if @kostmo could double check that'd be great